api/wasm/cpp/proxy_wasm_intrinsics.proto

```diff
@@ -1,7 +1,82 @@
 syntax = 'proto3';
 
+import "google/protobuf/any.proto";
 import "google/protobuf/struct.proto";
 
 message WKT {
   google.protobuf.Struct struct = 1;
 }
+
+message DataSource {
+  oneof specifier {
+    string filename = 1;
+    bytes inline_bytes = 2;
+    string inline_string = 3;
+  }
+}
+
+message GrpcService {
+  message EnvoyGrpc {
+    string cluster_name = 1;
+  }
+  message GoogleGrpc {
+    string target_uri = 1;
+    message SslCredentials {
+      DataSource root_certs = 1;
+      DataSource private_key = 2;
+      DataSource cert_chain = 3;
+    }
+    message GoogleLocalCredentials {
+    }
+    message Empty {
+    }
+    message ChannelCredentials {
+      oneof credential_specifier {
+        SslCredentials ssl_credentials = 1;
+        Empty google_default = 2;
```

Comment:
Why can't we use `google.protobuf.Empty google_default = 2;` here? Seems to work fine for me.

Reply:
Done.